compiler/main/driver.cpp

```diff
@@ -506,6 +508,24 @@ static void setupChplHome(const char* argv0) {
   }
 }
 
+#ifndef HAVE_LLVM
```

Comment:
Might be a little cleaner to stuff this `ifndef` into the the function and avoid the `ifndef` at the call site? No real preference here, so I'm fine leaving as-is.

Reply:
Did this.